File: scratch/inspect_2026_07.py
import openpyxl
import json

filepath = r"C:\Users\k0115\OneDrive\바탕 화면\2026-07월매입매출현황의 복사본.xlsx"
wb = openpyxl.load_workbook(filepath, data_only=True)
print("Sheet names:", wb.sheetnames)

for sheet in wb.sheetnames:
    ws = wb[sheet]
    print(f"\n==========================================")
    print(f"=== Sheet: {sheet} (rows={ws.max_row}, cols={ws.max_column}) ===")
    print(f"==========================================")
    for r in range(1, min(15, ws.max_row + 1)):
        row_vals = [ws.cell(row=r, column=c).value for c in range(1, min(15, ws.max_column + 1))]
        if any(v is not None for v in row_vals):
            print(f"Row {r:2d}:", [str(v) if v is not None else "" for v in row_vals])

File: scratch/inspect_2026_07.js
import fs from "fs";
import * as XLSX from "xlsx";

const filepath = "C:\\Users\\k0115\\OneDrive\\바탕 화면\\2026-07월매입매출현황의 복사본.xlsx";

try {
  const buf = fs.readFileSync(filepath);
  const workbook = XLSX.read(buf, { type: "buffer" });
  console.log("Sheet names:", workbook.SheetNames);

  for (const sheetName of workbook.SheetNames) {
    const sheet = workbook.Sheets[sheetName];
    const data = XLSX.utils.sheet_to_json(sheet, { header: 1 });
    console.log(`\n==========================================`);
    console.log(`=== Sheet: ${sheetName} (Rows: ${data.length}) ===`);
    console.log(`==========================================`);
    for (let i = 0; i < Math.min(15, data.length); i++) {
      if (data[i] && data[i].length > 0) {
        console.log(`Row ${i + 1}:`, data[i].slice(0, 15));
      }
    }
  }
} catch (e) {
  console.error("Error reading excel:", e);
}
